packages/gatsby-cli/package.json

```
@@ -28,7 +28,7 @@
     "source-map": "^0.5.7",
     "stack-trace": "^0.0.10",
     "update-notifier": "^2.3.0",
-    "yargs": "^11.1.0",
+    "yargs": "12.0.5",
```

Since this is also a major semver upgrade, are there any breaking changes we should resolve? Also, should we add a ^ as opposed to pinning the version?

---

```suggestion
    "yargs": "^11.1.0",
```
Would probably be better to not change the version at all in this PR, what do you think?

---

Just saw that scriptName is 12+